gravitl/netclient
# fix(GRA-1223): update nc gui settings page

TESTING STEPS
1. Go to netclient gui settings and check that all fields load properly
2. Check that no new bugs have been introduced
